tinygrad/codegen/kernel.py

```diff
@@ -669,16 +673,19 @@ def fix_st(warp_dims, tcd_dims, tcd_expand, pattern_1, pattern_2, st1):
             return st1.reshape(new_shape).simplify().permute(tuple(permaxis)).reshape(st1.shape).simplify()
 
           if self.opts.device in {"AMD", "HIP"}:
-            reduce_axes, upcast_axis = [0], [[(0, 16)], [(0, 16)], [(1, 8)]]
+            reduce_axes, upcast_axes = [0], [[(0, 16)], [(0, 16)], [(1, 8)]]
```

Comment:
If you want things like this rename, it should be in a separate PR. Always separate whitespace/refactor from functional changes.

Reply:
Okey, moved into separate PR as well